vault/data_source_azure_access_credentials_test.go

```diff
@@ -35,10 +34,6 @@ func TestAccDataSourceAzureAccessCredentials_basic(t *testing.T) {
 					resource.TestCheckResourceAttrSet("data.vault_azure_access_credentials.test", "lease_id"),
 				),
 			},
-			{
```

Comment:
is this test no longer needed? Seems like it was maybe flaky?

Reply:
I don't actually understand this test. I think possibly the source code that it was testing has since been changed. I can't find any error string in TFVP or vault that matches the regex used here.